scripts/publish_posthook.ts

```diff
@@ -11,6 +12,29 @@ if (args.length != 4) {
 	process.exit(1);
 }
 
+// copyDeclarationFilesToDestination copies the declaration files contained in
+// `src` into `dest` with their respective subfolder structure.
+// e.g.:
+//              src/x/y/z.d.ts -> dist/x/y/z.d.ts.
+function copyDeclarationFilesToDestination(src: string, dest: string) {
```

Comment:
```suggestion
function copyDeclarationFiles(src: string, dest: string) {
```
IMHO `ToDestination` is noise